From 1239f57e29e265860f2cfa2331a9d3344d23c743 Mon Sep 17 00:00:00 2001
From: MichaelGetu-git <michaelgetu21@gmail.com>
Date: Fri, 5 Dec 2025 23:52:04 +0300
Subject: [PATCH 08/12] fix: real time collaboration via services fixed

---
 client/src/TextEditor.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/client/src/TextEditor.js b/client/src/TextEditor.js
index 33fcd24..dc1eb5d 100644
--- a/client/src/TextEditor.js
+++ b/client/src/TextEditor.js
@@ -29,6 +29,7 @@ export default function TextEditor() {
 
     // Connect to collaboration service through API Gateway
     const s_socket = io(process.env.REACT_APP_API_URL || 'http://localhost', {
+      transports: ['websocket'],
       auth: {
         token: token
       }
-- 
2.44.0.windows.1

